<!-- The launch file is used to run the visual odometry -->

<launch>

    <arg name="use_rviz" default='true' doc="Launch rviz to visualize the map"/>

    <rosparam command="load" file="$(find stereo_visual_slam_main)/config/kitti_param.yaml" />

    <node name="run_vslam" pkg="stereo_visual_slam_main" type="run_vslam" output="screen"/>

    <group if="$(eval arg('use_rviz') == true)">
        <node name="rviz" pkg="rviz" type="rviz" args="-d $(find stereo_visual_slam_main)/config/kitti_config.rviz"/>
    </group>

</launch>